nengo/spa/spa.py

```diff
@@ -80,153 +76,20 @@ def __init__(self):
 
     def __init__(self, label=None, seed=None, add_to_container=None,
                  vocabs=None):
-        super(SPA, self).__init__(label, seed, add_to_container)
+        warnings.warn(DeprecationWarning(
+            "nengo.spa.SPA is deprecated. Use nengo.spa.Module instead."))
```

Comment:
I think it's cleaner because there is no reason to have two classes around basically doing the same thing.